### PR TITLE
Added libffi 

### PR DESCRIPTION
**Pull request by RealDekkia**
This library is a requirement for Borgbackup and need to be installed alongside for it to work

See forum for further discussion.

### CODE REVIEW

**Comment by neupsh** [comment]
@UnRAIDES any chance this gets merged soon? :)

**Comment by miquelbotanch** [comment]
Thank you ! Will be the Solution to be able to run borgbackup without messing up with user scripts at boot time.

**Comment by RealDekkia** [comment]
Library seems also to be a requirement for iotop: https://github.com/UnRAIDES/unRAID-NerdTools/issues/37#issue-1480051967

**Comment by skjnldsv** [comment]
@jsavargas @UnRAIDES any news? :)

File: packages/packages-desc [hunk 1]
```diff
@@ -59,6 +59,7 @@
  "lbzip2":"pthreads-based parallel bzip2\/bunzip2",
  "lftp":"A program that allows sophisticated ftp and http connections to other hosts. lftp can handle many file access methods including ftp, ftps, http, https, hftp, fish and file. Every operation in lftp is reliable, that is any non-fatal error is ignored and the operation is retried. lftp has shell-like command syntax. Has a built in mirror feature which can download or upload a whole directory tree. To learn about many more features of lftp, see  man lftp.",
  "libassuan":"This is the IPC library used by GnuPG 2.x, GPGME and a few other packages.",
+ "libffi":"A library used by Borgbackup.",
```
Comment (drewbitt) [inline packages/packages-desc:62]: It is also used by iotop

**Comment by drewbitt** [comment]
Unraid comes with a version of libffi, `ls /usr/lib64/libffi*`. It is just too new for BorgBackup/iotop. Since anything NerdTools puts in /boot/extras will not install with the existing newer version present, this would not by default work without additional scripting. Although I recognize its utility if it fixes borgbackup/iotop, NerdTools installing (older) packages installed already is odd.

**Comment by RealDekkia** [comment]
Of course It would make more sense to update borg to hopefully use the newer libffi.
(the current 1.1.15 is very old)

The latest slackware build I could find is 1.1.18: https://slackbuilds.org/repository/15.0/system/borgbackup/
But I could't get it to run at all and don't really have enough knowledge to fix that.

And even then this version might not be new enough.

**Comment by hapm** [comment]
Looks like this problem affects a lot of python programms, especially the ones using `ctypes`. This can be easily verified by starting python in a console and trying to `import ctypes`:

```
>>> import ctypes
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/ctypes/__init__.py", line 7, in <module>
    from _ctypes import Union, Structure, Array
ImportError: libffi.so.7: cannot open shared object file: No such file or directory
```

It affects `python` and `python3`!

**Comment by UnRAIDES** [comment]
Hi,
The libffi-3.4.4 is added for iotop. 
Borgbackup will be updated very soon.

Thank you.

**Comment by Mcklaren** [comment]
Hello,

Borgbackup is update on 2023.05.02 version.

If you need libffi please reopen this PR of new. I test the new Borgbackup and I think that it's not necessary.

Thank you.

**Pull request closed without merging**